src/index.js

```diff
@@ -1,14 +1,135 @@
-import _ from 'lodash';
```

Comment:
When adding or deleting a task from your to-do list its important that the tasks property index is always set to the value of the new array length. 

![Captura](https://user-images.githubusercontent.com/15807343/154734193-5dd50e4b-09d8-42f3-94d6-a60475c0ad5c.PNG)

As you can see in the image, After playing around adding, deleting, and editing tasks, the left task index is 3 when it should be 1. Kindly make the necessary changes to that your app works as mentioned 🔥

The mentioned issue with the indexes of your tasks is causing another one. If you create a single task and you remove it, works perfectly. If you create a bunch of tasks and then delete the first-created ones until only the last one is left and you try to delete it doesn't work properly. After the page is refreshed, the task is still there. I assume its something to do with the index not matching the one you think you are removing 👀